MandelaK/SendIT-2
# #[161709465] api get all orders

#### What does this PR do?
Added endpoints for getting all orders in our database

#### Description of Task to be completed?
We need endpoints to handle requests for when users want to know all the requests in our database.

#### How should this be manually tested?
Cloning the repository and installing all requirements in a virtual environment is the initial setup. From there we run the app and add some dummy data through postman and attempt retrieving it. We should get a 200 status code and a list of all deliveries appended to our database.

#### What are the relevant pivotal tracker stories?
#161709465
